src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -734,6 +803,130 @@ void Qgs3DMapCanvasWidget::onGpuMemoryLimitReached()
   mGpuMemoryLimitReachedReported = true;
 }
 
+void Qgs3DMapCanvasWidget::onPointCloudChangeAttributeSettingsChanged()
+{
+  const QString attributeName = mCboChangeAttribute->currentText();
+
+  if ( attributeName == QLatin1String( "Intensity" ) )
```

Comment:
There is no Enum for this?

Reply:
There's an Enum for laz attributes but it is only used privately in QgsLazDecoder.
It could be a good move to integrate it with QgsPointCloudAttribute and use it in all places, but this is beyond the scope of this PR